### DISCUSSION TITLE
Getting exported files for only one case

### DISCUSSION THREAD

**Discussion by thomas-saigre** [feelpp/feelpp#1834] [Ideas]
[Here is](https://gist.github.com/thomas-saigre/7302a753dd851bcb35cf80f886fb4d4d) a python script allowing you to synchronize the exported Feel++ files only for the case just executed (this avoids having to synchronize the whole `feel` folder from the remote server). Just modify lines 10 and 11 to have the right login / remote machine, then enter the command `sync-case --dir <the folder displayed at the end of the feelpp run>`.

By default this will sync into `<feel global_root>/<server>/<corresponding folder>` (obtained by reading the .feelppconfig) (it is also possible to change this using the options, some options are also available, type `sync-case --help` or look at the source to see them)

**Comment by prudhomm** [comment]
We could certainly integrate the tool into feelpp and install it along the other tools

**Comment by thomas-saigre** [comment]
See pull request #1917
